x11-packages/gnucash/build.sh

```diff
@@ -0,0 +1,17 @@
+TERMUX_PKG_HOMEPAGE="https://gnucash.org"
+TERMUX_PKG_DESCRIPTION="Personal and small-business financial-accounting software"
+# Licenses: GPL-2.0-or-later, GPL 2 or 3, public domain, special exception for
+# linking OpenSSL.
+TERMUX_PKG_LICENSE="custom"
```

Comment:
That license can be accurately described by `GPL-2.0-or-later`, with the additional nuance covered by the specific LICENSE file in the project repository.

```suggestion
TERMUX_PKG_LICENSE="GPL-2.0-or-later" # with OpenSSL linking exceptions
TERMUX_PKG_LICENSE_FILE="LICENSE"     # specified for additional nuance.
```

As an addendum, to go into detail why we need to specify the `TERMUX_PKG_LICENSE_FILE` here.
We ordinarily try to determine this automatically from a "common license filenames" list;
https://github.com/termux/termux-packages/blob/be9b63a19cc67e4c24cc2842e6a5430e59af36e2/scripts/build/termux_step_install_license.sh#L33-L55
However, we also try to reduce package size by symlinking to "generic" licenses (without project specific information) provided by the `termux-licenses` package.
The GPL (and variants) is ordinarily such a "generic" license.
As such, we need to specify a license file to ensure it copies in the one from the repo since there is additional nuance in this case.
https://github.com/termux/termux-packages/blob/be9b63a19cc67e4c24cc2842e6a5430e59af36e2/scripts/build/termux_step_install_license.sh#L8-L32